app/routes/index.ts

```diff
@@ -75,6 +75,7 @@ router.get('*', (req, res, next) => {
   if (env.NODE_ENV === 'development') {
     // Prevent proxy recursion
     const p = new URLSearchParams(req.params);
+    p.delete('0');
```

Comment:
What does this do?

Reply:
No clue. Removed it since it seemed to have no use, but mayb i'll spend some time to see where it came from:3

Reply:
Think line above should also be req.query, not ![params](http://expressjs.com/en/4x/api.html#req.params), which may be why this is happening. Ima change it